pkgs/racket-index/setup/scribble.rkt

```diff
@@ -1717,3 +1718,7 @@
     (build-path base root)]
    [else
     (reroot-path base root)]))
+
+
+(define (doc-pkg? doc)
+  (and (path->pkg (doc-src-file doc)) #t))
```

Comment:
There's already a `#:cache` argument in `path->pkg` for memoization, right?

Reply:
FWIW that `cache` argument just need to be an empty hash to start (and then not used for anything else). The docs aren't super clear on how to use that argument but almost every call to `path->pkg` (and other similar functions) should be using that argument.

Reply:
Ok, I updated that to use `#:cache`.